packages/typeless/src/Epic.ts

```diff
@@ -86,10 +86,14 @@ export class Epic {
         if (Array.isArray(result)) {
           return from(result);
         }
-        if (isAction(result) || result === null) {
+        if (isAction(result)) {
           return of(result);
         }
 
+        if (result === null) {
+          return of(null);
+        }
```

Comment:
Before this change, it is emitted error in test compiling.
Because tsc can't detect a `null` type in `strictNullChecks: false` env...